src/pages/community/faq.md

```diff
@@ -23,6 +23,21 @@ They'll take your code from the (macOS) clipboard, convert it, and paste it back
 #### What's the relation between Reason, BuckleScript and OCaml?
 See [here](/guide/javascript). Reason's a syntax for OCaml and supports all its features. BuckleScript compiles OCaml/Reason code into JavaScript.
 
+#### How does Reason (and BuckleScript) compare to Elm
+* Reason is a general-purpose language that can target node.js, native code and utilize both the opam and npm ecossytems. Elm is more opininated and focused, aiming at improving the experience in its own niche and ecosystem, at the expense of flexibility.
+* Elm has better error messages, though Reason is closing in fast.
+* BuckleScript has a very simple, direct and flexible FFI system, and generates highly readable js code, enabling gradual adoption and taking advantage of the full npm ecosystem. Elm's approach is more strict, emphasizing safety and as a consequence has an FFI system that is comparatively involved.
+* Elm is pure, while Reason is "pragmatic" (take that as you wish :)).
+* OCaml has had 25 or so years to mature, and has an active academic base of contributors that keep it close to the forefront of progamming langauge development, while Elm is relatively new and unstable, but moves faster and has less legacy cruft.
+* Reason has first-class support for React, while Elm is focused solely on "The Elm Architecture" (TEA). Reason does however also have [a TEA implementation](https://github.com/OvermindDL1/bucklescript-tea) that does a surprisingly good job of mimicing its muse.
+
+#### How does Reason (and BuckleScript) compare to PureScript
+* Like Elm, PureScript is relatively new and unstable, more so than Elm. It also targets the JavaScript niche, but unlike Elm it doesn't focus solely on the front-end.
+* Like BuckleScript, PureScript compiles to highly readable JavaScript code, and has a significantly simpler FFI system than Elm, though its purity does impose some limitations.
+* PureScript also targets the bower ecosystem instead of npm.
```

Comment:
this isn't necessarily true anymore, but also isn't as much an issue for purescript. it isn't use bower to replace npm, it just doesn't allow nested resolution of modules so I'm not sure that it matter for the conversation?

Reply:
Huh, TIL, do you have a source that expands on that?